lib/compiler/compileSolcjs.ts

```diff
@@ -3,7 +3,7 @@ import {promisify} from 'util';
 import {readFileContent, isDirectory, relativePathToWorkingDir} from '../utils';
 import {Config} from '../config/config';
 import {buildInputObject} from './buildUitls';
-import { ImportFile } from '@resolver-engine/imports';
+import {ImportFile} from '@resolver-engine/imports';
```

Comment:
Good stuff on fixing formatting. In the future, this could go as a separate PR.

Reply:
https://github.com/EthWorks/Waffle/pull/148 <- future is now :sunglasses:

Reply:
Separate PR is a good idea, but I'd rather prefer to merge it before this one.

Reply:
👌